bakerytest/bakerytest.go

```diff
@@ -21,6 +23,51 @@ type Discharger struct {
 	server *httptest.Server
 }
 
+var skipVerify struct {
+	mu            sync.Mutex
```

Comment:
you could possibly use sync/atomic instead  of a lock here , but  it isn't important.